include/aspect/utilities.h

```diff
@@ -392,4 +392,27 @@ namespace aspect
   }
 }
 
+namespace dealii
+{
+  namespace Utilities
+  {
+    namespace MPI
+    {
+      /**
+       * Perform an MPI sum over the quantities in the given Tensor<1,dim>
+      */
+      template<int dim>
+      Tensor<1,dim> 
+      reduce_tensor(const Tensor<1,dim> &local, const MPI_Comm &mpi_communicator);
+
+      /**
+       * Perform an MPI sum over the quantities in the given SymmetricTensor<2,dim>
+       */
+      template<int dim>
+      SymmetricTensor<2,dim> 
+      reduce_symmetric_tensor(const SymmetricTensor<2,dim> &local, const MPI_Comm &mpi_communicator);
```

Comment:
I thought of these as going into deal.II's `dealii::Utilities::MPI` where the other `sum` functions already are. I think `sum` would actually be a descriptive name for the function.